lib/arrays.md

```diff
@@ -5,44 +5,44 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is an Array?
 
-Replace me with your response...
+An array is a collection of information stored as a list. 
 
 # What are some examples of information that would be Arrays as opposed to some other data type?
 
-Replace me with your response...
+Considering that arrays can be comprised of any data type, any information that tends to be listed or stored as a database would work well as an array, such as a table of contents of a book or names of students in a class. 
 
 # What is one way, using Ruby, to retrieve the 6th element in an Array? How about the 8th element? What happens if you try to retrieve the value of the _9999th_ element (or some element that doesn't exist in the array)?
 
-Replace me with your response...
+One way to retrieve an element of an Array would be to use the element's index. For example for the 6th element in an array it would be "array[5]". As well as for the 8th, it would be "array[7]. If you try to retrieve an element that doesn't exist in the array, ruby will return nothing. 
```

Comment:
Right on. Just a quick tip, we call "nothing" in Ruby `nil` to distinguish it from other kinds of "nothingness" like `""` (a blank string) or `0`. No revision needed–just an FYI.